client/app/visualizations/counter/utils.test.js

```diff
@@ -0,0 +1,192 @@
+import { getCounterData } from './utils';
+
+let dummy;
+
+describe('Visualizations -> Counter -> Utils', () => {
+  beforeEach(() => {
+    dummy = {
+      rows: [
+        { city: 'New York City', population: 18604000 },
+        { city: 'Shangai', population: 24484000 },
+        { city: 'Tokyo', population: 38140000 },
+      ],
+      options: {},
+      visualisationName: 'Visualisation Name',
```

Comment:
`visualisation` -> `visualization` :slightly_smiling_face:

Reply:
Nice catch and both forms are correct in terms of grammar if it's what you meant. These are just test dummies; might as well be _foo name_. Please feel free to rename to whatever you prefer.

Reply:
It was actually to keep it as is in the rest of the codebase, but NP, I'll revisit those tests in #4392.